### PR TITLE
Report improvements

### PR DESCRIPTION
**Pull request by EricLindskog**
Improved the structure of the html and made the info box move on resize.

### CODE REVIEW

File: plugin/mutate/source/dextool/plugin/mutate/backend/report/html/package.d [hunk 1]
```diff
@@ -170,24 +170,31 @@ struct FileIndex {
...
         foreach (const s; ctx.span.toRange) {
             if (s.tok.loc.line > lastLoc.line) {
                 lastLoc.column = 1;
             }
-
             auto meta = MetaSpan(s.muts);
 
             foreach (const i; 0 .. max(0, s.tok.loc.line - lastLoc.line)) {
-                root.addChild("br");
+                //root.addChild("br"); Not needed when grouping locs in divs
```
Comment (joakim-brannstrom) [inline plugin/mutate/source/dextool/plugin/mutate/backend/report/html/package.d]: Remove the line

File: plugin/mutate/source/dextool/plugin/mutate/backend/report/html/package.d [hunk 2]
```diff
@@ -170,24 +170,31 @@ struct FileIndex {
         auto lastLoc = SourceLoc(1, 1);
 
         auto root = ctx.doc.mainBody;
-        root.addChild("span", "1:").addClass("line_nr");
+        //Div for line 1
+        auto div = root.addChild("div").setAttribute("id",format("%s-%d","line",(1)));
```
Comment (joakim-brannstrom) [inline plugin/mutate/source/dextool/plugin/mutate/backend/report/html/package.d]: Deviates from the style of the project.
run dfmt on the changes files.
```sh
# assuming you are in the root of dextool. the style config is there (.editorconfig)
dub run dfmt -- -i plugin/mutate.....
```

File: plugin/mutate/source/dextool/plugin/mutate/backend/report/html/package.d [hunk 3]
```diff
@@ -170,24 +170,31 @@ struct FileIndex {
...
-
             auto meta = MetaSpan(s.muts);
 
             foreach (const i; 0 .. max(0, s.tok.loc.line - lastLoc.line)) {
-                root.addChild("br");
+                //root.addChild("br"); Not needed when grouping locs in divs
                 // force a newline in the generated html to improve readability
                 root.appendText("\n");
-                root.addChild("span", format("%s:", lastLoc.line + i + 1)).addClass("line_nr");
+                //New line reached, add new div
```
Comment (joakim-brannstrom) [inline plugin/mutate/source/dextool/plugin/mutate/backend/report/html/package.d]: I think it is a redundant comment considering line 183.

**Comment by joakim-brannstrom** [comment]
The failed CI (Azure) is unrelated to your changes. It failed downloading a packet during the installation.
5/6 targets is green so good job.

**Comment by EricLindskog** [comment]
Closing pull request as it contained changes that are not complete.

**Pull request closed without merging**